### PR TITLE
Release 02/26/2023-03/05/2023

### PR DESCRIPTION
**Pull request by miwithro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+* Starting with Kubernetes 1.26:
+  * HostProcess Containers will be GA
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Preview Features
```
Comment (CocoWang-wql) [inline CHANGELOG.md:22]: [Trusted Access](https://learn.microsoft.com/en-us/azure/aks/trusted-access-feature) is public preview now.

**Comment by qpetraroia** [comment]
Release 02/26/2023-10/10/2022

Title has the wrong dates

**Comment by miwithro** [comment]
@qpetraroia fixed.

@CocoWang-wql fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+  * [Azure CNI Overlay](https://learn.microsoft.com/azure/aks/azure-cni-overlay) Public Preview is now available in ALL Azure Public Cloud Regions.
+  * [Trusted Access](https://learn.microsoft.com/en-us/azure/aks/trusted-access-feature) is now in Public Preview.
+* Bug Fix
+  * Fixed issue with Linux node outbound connectivity failing due to HTTP_PROXY/HTTPS_PROXY not fully respected.
+  * Fix to allow a stopped AKS cluster to rotate certificates/
```
Comment (qpetraroia) [inline CHANGELOG.md:30]: Fix to allow a stopped AKS cluster to rotate certificates/ 
 
 There is a "/" here

**Comment by olsenme** [comment]
@miwithro , could you please mention this in 1.26 announcements: 
Starting Kubernetes v1.26 two in-tree driver persistent volumes won't be supported in AKS : kubernetes.io/azure-disk, kubernetes.io/azure-file.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+* Behavior Changes
+  * Increased qps limits and worker threads for CSI driver on azuredisk v2.
+  * the token credential will gradually be rotated. it shall not incur any impact since kubeconfig has the client certificate. should you see any issue, call az aks get-credentials again.
+  * For customers using the Web App Routing add-on (Preview), we added an "identity" field in the API response exposing the managed service identity creates by the add-on. You can grant that identity permissions to manage other Azure resources used by the add-on, such as Azure DNS and Azure Key Vault.
+  * Extended cluster tag support to Standard Load Balancer and Public IP.
+  * Bumped the memory limit for the Container Insights Add-on for Windows to 1Gb.
+  * Customer applied tags on Azure cloud provider managed resources (LB, publicIp, NSG, PLS, etc) under node resource group would be overwritten allowing the tags to removed from the resources when requested.
+* Component Updates
+  * Image Cleaner addon updated to version [v1.0.0](https://github.com/Azure/eraser/releases/tag/v1.0.0)
```
Comment (CocoWang-wql) [inline CHANGELOG.md:39]: Pls remove it as this change is reverted from 0305 release.

File: CHANGELOG.md (hunk 2 above)
Comment (CocoWang-wql) [inline CHANGELOG.md:30]: Fix to allow a stopped AKS cluster to rotate certificates/ -> pls remove it as this change has been reverted from 0305 release.

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,49 @@
...
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+  * Two in-tree driver persistent volumes won't be supported in AKS : kubernetes.io/azure-disk, kubernetes.io/azure-file.
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Preview Features
+  * [Azure CNI Overlay](https://learn.microsoft.com/azure/aks/azure-cni-overlay) Public Preview is now available in ALL Azure Public Cloud Regions.
```
Comment (wangyira) [inline CHANGELOG.md:27]: this disclosure is from a while ago, is there another public preview for CNI overlay?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,49 @@
...
+  * [Trusted Access](https://learn.microsoft.com/en-us/azure/aks/trusted-access-feature) is now in Public Preview.
+* Bug Fix
+  * Fixed issue with Linux node outbound connectivity failing due to HTTP_PROXY/HTTPS_PROXY not fully respected.
+  * Fix to allow a stopped AKS cluster to rotate certificates.
+* Behavior Changes
+  * Customer applied tags on Azure cloud provider managed resources (LB, publicIP, NSG, PLS, etc) under node resource group will now be overwritten when a task to adjust the current state to match the desired state is invoked. Please follow [AKS docs](https://learn.microsoft.com/en-us/azure/aks/use-tags)
+and apply the tags on the cluster if the tags are required on the AKS managed resources.
+  * Increased qps limits and worker threads for CSI driver on azuredisk v2.
+  * the token credential will gradually be rotated. it shall not incur any impact since kubeconfig has the client certificate. should you see any issue, call az aks get-credentials again.
```
Comment (wangyira) [inline CHANGELOG.md:36]: NIT: "the" and "it" are not capitalized

**Review by wangyira — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (miwithro) [reply CHANGELOG.md:28]: @wangyira this is announcing that the Public Preview is now available in ALL regions, where as it initially was not.

**Review by wangyira — APPROVED** [review]

**Pull request merged**